tests/testthat/indention_multiple/overall-in.R

```diff
@@ -1,30 +1,31 @@
-#'this function does
+#' this function does
```

Comment:
oops, actually not but we have another test case to cover this so I leave it for now if that's ok